src/config/delegates.json

```diff
@@ -1,4 +1,16 @@
 {
+ "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9": {
+    "name": "ReFiCol4🌎",
+    "address": "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9",
+    "logoUri": "/logos/validators/ReFi-Col.jpg",
+    "date": "2024-05-01",
+    "links": {
+      "website": "https://linktr.ee/reficolombia",
+      "twitter": "https://twitter.com/reficolombia”
+    },
+    "interests": [“”ReFi”, “Community”, Transparency", "Accountability", "Good Vibes"],
```

Comment:
The quote characters are wrong in several places here. This is invalid JSON and won't parse. Please validate your submission: https://jsonlint.com/

Reply:
Solved